src/main/java/picard/util/IntervalListTools.java

```diff
@@ -427,7 +427,8 @@ protected int doWork() {
             errorMsgs.add("BREAK_BANDS_AT_MULTIPLES_OF must be greater than or equal to 0.");
         }
 
-        if (CollectionUtil.makeSet(Action.CONCAT, Action.UNION, Action.INTERSECT).contains(ACTION)) {
+        if (CollectionUtil.makeSet(Action.CONCAT, Action.UNION, Action.INTERSECT).contains(ACTION)
```

Comment:
Instead of `CollectionUtil.makeSet()` use `EnumSet.of()`. Also a static set like this should be in a static final field.